apps/libsignpost/signbus_io_interface.c

```diff
@@ -129,6 +121,19 @@ static void i2c_master_slave_callback(
     } else if (callback_type == TOCK_I2C_CB_SLAVE_READ_COMPLETE) {
         signbus_iterate_slave_read();
     }
+}*/
+
+void signbus_io_slave_write_callback(int len_or_rc);
```

Comment:
Without it, the build system throws an error: "no previous prototype for 'signbus_io_slave_write_callback'"